.claude/skills/ui-ux-pro-max/scripts/design_system.py

```
@@ -0,0 +1,1067 @@
+    layout = {}
+    spacing = {}
+    typography = {}
+    colors = {}
+    components = []
+    unique_components = []
+    recommendations = []
+    
+    # Extract style-based overrides
+    if style_results:
+        style = style_results[0]
+        style_name = style.get("Style Category", "")
```

Ruff (0.14.14)

[error] Local variable `style_name` is assigned to but never used

Remove assignment to unused variable `style_name`

(F841)